fuels-signers/src/util.rs

```diff
@@ -0,0 +1,80 @@
+#[allow(missing_docs)]
+/// Testing utilities
+pub mod test_helpers {
```

Comment:
Since these are publically exported, I'd recommend putting these under a feature flag.

```suggestion
#[cfg(feature = "test-helpers")]
pub mod test_helpers {
```

Reply:
That's what I did initially, but we have doc tests (https://github.com/FuelLabs/fuels-rs/pull/110/files#diff-dabbabe6828f1fb7638ca0ace7dd311af93e4f83fc42d234246fdef19d086cf0R140) that's importing these `test_helpers`, and in doc level examples you can't import anything behind a feature flag without doing very hacky things: https://users.rust-lang.org/t/doctests-that-require-a-non-default-feature-is-it-possible/29529/4

Reply:
I think you should be able to just simply enable all the deps and features you need in the dev-deps to get the rust docs working..

e.g. 
```
[dev-dependencies]
fuels-rs = { path = ".", version = "0.x", features = ["test-helpers"] }
```

Reply:
Oh, I see, TIL. That works!